cmake_format/vscode_extension/src/extension.ts

```diff
@@ -16,31 +19,17 @@ export function activate(context: vscode.ExtensionContext) {
   let disposable = vscode.languages.registerDocumentFormattingEditProvider(
     { scheme: 'file', language: 'cmake' }, {
     provideDocumentFormattingEdits(document: vscode.TextDocument): vscode.TextEdit[] {
-      var fs = require('fs')
-      var path = require('path')
       var config = vscode.workspace.getConfiguration('cmakeFormat');
-      var exePath = config.get("exePath");
+      var exePath = config.get<string>("exePath", "cmake-format");
+      var args = [document.uri.fsPath].concat(config.get<string[]>("args", []));
+      var opts: any = {};
+      var cwd = config.get<string>("cwd");
 
-      var args = config.get<string[]>("args", [])
```

Comment:
This logic is important. Please add it back. Please test your change with `"argv"` set to `["--config", "/path/to/your/config"]` to ensure this issue is addressed.

Reply:
What do you mean "add it back"? I haven't removed the logic and I've tested it with a custom config file. The code simply changed to have the path to the file as the first argument as passing the contents via `stdin` wasn't working.
Updated snippet: `var args = [document.uri.fsPath].concat(config.get<string[]>("args", []));`

Reply:
Hm... When I looked at the delta it showed a removal of the `--` logic. Was that not the case? That commit seems to be gone so I can't double check. Maybe it was a github display error then?

Reply:
Ah yeah, that logic was removed since it was causing the argument parser to fail for `cmake-format`. I think the confusion was on my end because your initial comment seemed to suggest I had removed any ability to pass arguments entirely. Was just pointing out that I didn't.